docs: align portal iam doc structure with trg 1 (#207)
eclipse-tractusx/portal-iam: docs/README.md

### 2. [Developer Process](./admin/dev-process/)

- [Dev Flow](./admin/dev-process/Dev%20Flow.md): This document outlines the development flow, using a git-based diagram to show the process of feature development, bug fixes, and release candidates. 
- [How to Contribute](./admin/dev-process/How%20to%20contribute.md): Steps for contributing to the project.

### 3. [Release Process](./admin/release-process/)

- [Release Process](./admin/release-process/Release%20Process.md) :  This document explains the release process, detailing steps from creating a release branch, updating versions, and generating changelogs to merging release candidates into the main branch. It follows semantic versioning practices and provides links to automated workflows used in the release cycle.

### 4. [Known-Knowns](./admin/known-knowns/)

The `known-knowns` folder highlights known limitations, issues, or considerations regarding the Portal IAM:

- [Known-Knowns](./admin/known-knowns/Known-Knowns.md): A list of known issues and solutions.

### [Consultation Documentation](./consultation/)

> ### 1. [Charts](./consultation/charts/)
> 
> The `charts` folder contains Helm charts for deploying the Portal IAM system. Refer to the following for detailed usage:
> 
> - [Portal IAM Helm Chart](./consultation/portal-iam-helm-chart.md): Instructions on using the Helm chart for deploying Portal IAM on Kubernetes.

Reviewer: ```suggestion
### 1. [charts](./consultation/charts/) and [environments](./consultation/environments/)

The `charts` and `environments` folder contain an **example** helm chart and configuration for a potential deployment improvement discussed during workshops.

- [Portal IAM Helm Chart](./consultation/portal-iam-helm-chart.md): instructions on using this **example** helm chart.
```

### 2. [Workshops](./consultation/workshops/)

> The `workshops` folder includes practical resources for learning and deploying Portal IAM:
> 
> - [Workshop 20230927](./consultation/workshops/workshop-20230927.md): Materials and steps covered during the September 27, 2023 workshop.
> - [Workshop 20231005](./consultation/workshops/workshop-20231005.md): Materials and steps from the October 5, 2023 workshop.

Reviewer: ```suggestion
The `workshops` folder includes minutes and topics for two workshops:

- [Workshop 20230927](./consultation/workshops/workshop-20230927.md): topics covered during the session of September 27, 2023 workshop.
- [Workshop 20231005](./consultation/workshops/workshop-20231005.md): topics covered during the session of October 5, 2023 workshop.
```

> ### 3. [Consultation Document](./consultation/consultation.md)
> - Key architectural consultation notes and decisions.

Reviewer: ```suggestion
### 3. [Consultation Document](./consultation/consultation.md)

- Consultation notes and decisions.
```
